includes/wc-template-functions.php

```diff
@@ -2456,7 +2456,7 @@ function wc_display_item_meta( $item, $args = array() ) {
 		) );
 
 		foreach ( $item->get_formatted_meta_data() as $meta_id => $meta ) {
-			$value = $args['autop'] ? wp_kses_post( $meta->display_value ) : wp_kses_post( make_clickable( trim( strip_tags( $meta->display_value ) ) ) );
+			$value = $args['autop'] ? wp_kses_post( $meta->display_value ) : wp_kses_post( make_clickable( trim( $meta->display_value ) ) );
```

Comment:
@WPprodigy we can keep br tags, this regex is just to revert a little what `wpautop()` does.